File: src/main/java/org/jukeboxmc/util/ServerKiller.java
package org.jukeboxmc.util;

import org.jukeboxmc.logger.Logger;

import java.util.concurrent.TimeUnit;

/**
 * @author LucGamesYT
 * @version 1.0
 */
public class ServerKiller extends Thread {

    private final Logger logger;

    public ServerKiller( Logger logger ) {
        this.logger = logger;
    }

    @Override
    public void run() {
        try {
            sleep( TimeUnit.SECONDS.toMillis( 3 ) );
        } catch ( InterruptedException e ) {
            throw new RuntimeException( e );
        }
        System.exit( 1 );
        this.logger.info( "Server shutdown successfully!" );
    }
}

File: JukeboxMC-Server/src/main/kotlin/org/jukeboxmc/server/util/ServerKiller.kt
package org.jukeboxmc.server.util

import org.jukeboxmc.api.logger.Logger
import java.util.concurrent.TimeUnit
import kotlin.system.exitProcess

class ServerKiller(private val logger: Logger) : Thread() {

    override fun run() {
        try {
            sleep(TimeUnit.SECONDS.toMillis(3))
        } catch (e: InterruptedException) {
            throw RuntimeException(e)
        }
        logger.info("Server shutdown successfully!")
        exitProcess(1)
    }
}